src/test/SessionManager.spec.ts

```diff
@@ -70,12 +85,11 @@ describe('SessionManager', () => {
           null,
           'access_token'
         )
-      ).rejects.toEqual(
-        new NoSessionStateError(
-          responseStatus,
-          process.env.SERVER_URL as string,
-          'logUrl'
-        )
+      ).resolves.toEqual('idle')
+
+      expect((process as any).logger.info).toHaveBeenCalledTimes(1)
```

Comment:
I'd assert on the number of times `axios.get` or `requestClient.get` was called (and the parameters with which it was called) here rather than just `logger`.

Reply:
The purpose of this test is to verify that regardless of how many times SAS server responded with status code 304, `@sasjs\adapter` logged `NoSessionStateError` only one time.

Reply:
just added one more assertion